src/Pages/CreatorPage.js

```diff
@@ -31,18 +40,17 @@ function CreatorPage() {
   useEffect(getPosts, []);
```

Comment:
Oh no, I think this may be another improperly merged naming issue! When some of the devs first started out they directly referenced the onboarding code and copied function names so that's why the function pulling an author is named 'getPosts'. Can you rename them to be more connotative of the functionality?